### ISSUE TITLE
support dnf system upgrades and notify users when a new release is available

### ISSUE THREAD

**Issue by rahulsundaram** [manatools/dnfdaemon#26]
Based on discussions at

https://lwn.net/Articles/773577/

**Comment by Conan-Kudo** [comment]
I'm not sure. I use KDE, so I never really used it. Unfortunately, Plasma Discover doesn't support this either. :(

**Comment by jcpunk** [comment]
Currently gnome-software uses https://github.com/GNOME/gnome-software/tree/master/plugins/fedora-pkgdb-collections  which checks https://admin.fedoraproject.org/pkgdb/api/collections

**Comment by rahulsundaram** [comment]
So now that we know that there is an API available, can upgrade support be added?

**Comment by Conan-Kudo** [comment]
Yeah, I can look into this. I'll need to figure out how to emulate the system-upgrade mechanism...

**Comment by Woi** [comment]
While I'd love to have this feature, a simple notification would already be a great improvement. I thought about something like:
* There is a newer release of Fedora available. To upgrade follow directions on https://fedoraproject.org/wiki/DNF_system_upgrade
* This Fedora release has reached EOL, and therefore don't receive any updates any more. This includes security fixes. It's strongly suggested to upgrade to a supported release of Fedora, following directions https://fedoraproject.org/wiki/DNF_system_upgrade

I could support by translating necessary messages to German

**Comment by Nokia808** [comment]
Really needed this feature if added will make dnfdragora better ...

**Comment by Conan-Kudo** [comment]
My understanding is that the API that GNOME Software uses now is deprecated and there is supposed to be a replacement for it. Also, we'd probably need to define an interface that multiple distributions can implement...
